src/main/kotlin/lotto/domain/LottoRank.kt

```diff
@@ -0,0 +1,17 @@
+package lotto.domain
+
+enum class LottoRank(val matchCount: Int, val reward: Long) {
+    FIRST(6, 2000000000L),
```

Comment:
작은 팁으로 언더스코어를 사용하여 가독성을 높일수 있습니다 😄 
```suggestion
    FIRST(6, 2_000_000_000L),
```

Reply:
오 이런 꿀팁이 있었군요 감사합니다 🙇